src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -197,13 +198,27 @@ public HttpResponse executeRequest(HttpUriRequest uriRequest, Header... header)
 			HttpResponse res = new HttpResponse(httpClient.execute(uriRequest));
 			log.debug(res.getBody());
 			return res;
+		} catch (final Exception e) {
+			throw new RuntimeException(e);
```

Comment:
Now that `Exception` is being thrown with the existing implementation, why does it need to be converted to `RuntimeException`?

Reply:
`httpClient.execute(...)` can still throw IOException